mtrf/model.py

```diff
@@ -363,12 +389,17 @@ def test(
                     "Hyperparameter optimization",
                     verbose=verbose,
                 ):
+                    cov_xx_this = None
```

Comment:
This feels not very informative, maybe cov_xx_train?

Reply:
Agree totally!